conans/client/printer.py

```diff
@@ -106,6 +106,7 @@ def _print(it_field, show_field=None, name=None, color=Color.BRIGHT_GREEN):
             _print("recipe", name="Recipe", color=None)
             if show_revisions:
                 _print("revision", name="Revision", color=None)
+                _print("package_revision", name="Package revision", color=None)
```

Comment:
Printing the package revision to screen too

Reply:
For the `assertNotIn` I did is the test not worth it ?

Reply:
The problem is that it was asserting that the revisions are not defined, the opposite of what we want. We want the revisions to be part of the information. That test now checks for ``self.assertEqual()`` to check that it is both there and has a value, instead of checking that it is not defined.